### ISSUE TITLE
python output issue with Chinese characters

### ISSUE THREAD

**Issue by Seraphli** [rgbkrk/atom-script#613]
While I using atom to write python 3.4 program, I try to print a string containing Chinese characters.
But the output show "我的名字是" as  "�ҵ�������".
I run the script in the terminal and everything is fine.
So I think it might be something wrong with encoding or font.
I try to use # -*-coding:utf-8 -* ,but it doesn't work.
So I post here to seek helps.
I'm using atom 1.0.19 on Windows 8.1

**Comment by tlongzou** [comment]
I also have this problem and I use CoffeeScript.

**Comment by Ansen** [comment]
I also encountered this problem，
my code :

    #!/usr/bin/python
    # -*- coding: utf-8 -*-
    print ("测试")

when press `ctrl+shift+b`
http://ww2.sinaimg.cn/large/867ecea7gw1ezd4ns9rocj20ei04cdft.jpg
![](http://ww2.sinaimg.cn/large/867ecea7gw1ezd4ns9rocj20ei04cdft.jpg)

**Comment by Seraphli** [comment]
Yeah, this problem finally made me stop using any Chinese character in the atom.

**Comment by rgbkrk** [comment]
Weird. This is working for me:

![screenshot 2015-12-29 11 01 54](https://cloud.githubusercontent.com/assets/836375/12038183/8f9efb8e-ae1b-11e5-90f2-d532b020af1c.png)

Ah, and you're both on Windows. I'm on OS X and unsure how to diagnose this for Windows.

Can you share what's in your `process.env`, which you can get by running:

```
console.log(process.env)
```

Make sure to scrub anything you don't want seen. I mostly care about, e.g. `LC_ALL`. There may be others.

**Comment by Seraphli** [comment]
Wow, I test today and version 3.2.0 works fine. I haven't test this after update. If there is no more issue, I should close this.
